.changeset/slimy-melons-accept.md

```diff
@@ -0,0 +1,6 @@
+---
+'@keystone-ui/fields': patch
+'@keystone-ui/options': patch
+---
+
+Patch to types based on changes to @types/react-select
```

Comment:
```suggestion
Patched types based on changes to @types/react-select.
```

Minor tweak for project wide consistency. The content of changesets ends up in our changelogs, so we try to write them as
 1) In past tense.
 2) As full sentences (e.g. full stop).